docs/release-notes/_snippets/9.1.4/index.md

```diff
@@ -0,0 +1,21 @@
+## 9.1.4 [elastic-agent-release-notes-9.1.4]
+
+
+### Features and enhancements [elastic-agent-9.1.4-features-enhancements]
+
+* Add the dockerstats OpenTelemetry receiver. [#9913](https://github.com/elastic/elastic-agent/pull/9913) [#9924](https://github.com/elastic/elastic-agent/pull/9924) [#9925](https://github.com/elastic/elastic-agent/pull/9925) [#9873](https://github.com/elastic/elastic-agent/pull/9873) [#9364](https://github.com/elastic/elastic-agent/pull/9364)
+* Bump kube-stack Helm Chart to 0.9.1 and enable the cluster collector. [#9913](https://github.com/elastic/elastic-agent/pull/9913) [#9924](https://github.com/elastic/elastic-agent/pull/9924) [#9925](https://github.com/elastic/elastic-agent/pull/9925) [#9873](https://github.com/elastic/elastic-agent/pull/9873) [#9535](https://github.com/elastic/elastic-agent/pull/9535)
+* Enhanced loggers for easier debugging of upgrade related issues. [#9913](https://github.com/elastic/elastic-agent/pull/9913) [#9924](https://github.com/elastic/elastic-agent/pull/9924) [#9925](https://github.com/elastic/elastic-agent/pull/9925) [#9873](https://github.com/elastic/elastic-agent/pull/9873) [#9689](https://github.com/elastic/elastic-agent/pull/9689) [#9536](https://github.com/elastic/elastic-agent/issues/9536)
+
+
+### Fixes [elastic-agent-9.1.4-fixes]
+
+* Redact secrets from pre-config, computed-config, components-expected, and components-actual files in diagnostics archive. [#9560](https://github.com/elastic/elastic-agent/pull/9560)
+* Retry service start command upon failure with 30-second delay. [#9313](https://github.com/elastic/elastic-agent/pull/9313)
+* Fix reporting of scheduled upgrade details across restarts and cancels. [#9562](https://github.com/elastic/elastic-agent/pull/9562) [#8778](https://github.com/elastic/elastic-agent/issues/8778)
+* Enable root user to re-enroll unprivileged agent for mac and linux. [#9603](https://github.com/elastic/elastic-agent/pull/9603) [#8544](https://github.com/elastic/elastic-agent/issues/8544)
```

Comment:
[#9603](https://github.com/elastic/elastic-agent/pull/9603) was closed automatically because it was moved to another branch.  Should it be included?